### DISCUSSION TITLE
Sorting Algorithm

### DISCUSSION THREAD

**Discussion by yami03** [smh-algorithms/algorithms#21] [도전 제안]
* Bubble Sort: https://www.codewars.com/kata/56b97b776ffcea598a0006f2 
* Insertion Sort
* Selection Sort
* Quick Sort
* Merge Sort

### 참고: 
* Bubble Sort: https://www.geeksforgeeks.org/bubble-sort/
* Insertion Sort: https://www.geeksforgeeks.org/insertion-sort/
* Selection Sort: https://www.geeksforgeeks.org/selection-sort/ 
* Quick Sort: https://www.geeksforgeeks.org/quick-sort/
* Merge Sort: https://www.geeksforgeeks.org/merge-sort/
* https://visualgo.net/en/sorting

**Comment by findawayer** [comment]
소트 강화주간 좋아요
